docs/rfcs/0002-layer-architecture.md

```diff
@@ -0,0 +1,328 @@
+
+<!-- Interoperability -->
+[Openness]:                   0001-high-level-architecture.html#subvalue-openness
+[De-facto standards]:         0001-high-level-architecture.html#subvalue-de-facto-standards
+[Declarative management]:     0001-high-level-architecture.html#subvalue-declarative-management
+[Consistency]:                0001-high-level-architecture.html#subvalue-consistency
+
+<!-- Accessibility / Reproducibility -->
+[Open source]:                0001-high-level-architecture.html#subvalue-open-source
+[Common off-the-shelf parts]: 0001-high-level-architecture.html#subvalue-common-off-the-shelf-parts
+[3D printed parts]:           0001-high-level-architecture.html#subvalue-3d-printed-parts
+[Reproducible PCBs]:          0001-high-level-architecture.html#subvalue-reproducible-pcbs
+[Documentation]:              0001-high-level-architecture.html#subvalue-documentation
+
+<!-- Modularity / Compatibility -->
+[Modular design]:             0001-high-level-architecture.html#subvalue-modular-design
+[Raspberry Pi compatibility]: 0001-high-level-architecture.html#subvalue-raspberry-pi-compatibility
+[Portability]:                0001-high-level-architecture.html#subvalue-portability
+[Loose coupling]:             0001-high-level-architecture.html#subvalue-loose-coupling
+
+<!-- Transparency -->
+[Observability]:              0001-high-level-architecture.html#subvalue-observability
+[Debuggability]:              0001-high-level-architecture.html#subvalue-debuggability
+[Energy monitoring]:          0001-high-level-architecture.html#subvalue-energy-monitoring
+
+<!-- Maintainability / Upgradability -->
+[Hot swappability]:           0001-high-level-architecture.html#subvalue-hot-swappability
+[Upgradability]:              0001-high-level-architecture.html#subvalue-upgradability
+[Auto-upgradability]:         0001-high-level-architecture.html#subvalue-auto-upgradability
+[One-time hardware setup]:    0001-high-level-architecture.html#subvalue-one-time-hardware-setup
+
+<!-- Affordability -->
+[Sensible rack cost]:         0001-high-level-architecture.html#subvalue-sensible-rack-cost
+
+<!-- User goals -->
+[Utilize Kubernetes]:         0001-high-level-architecture.html#user-goal-kubernetes
+[Fast reconfiguration]:       0001-high-level-architecture.html#user-goal-fast-reconfiguration
+[Secure updates]:             0001-high-level-architecture.html#user-goal-secure-updates
+[Zero-trust network boot]:    0001-high-level-architecture.html#user-goal-zero-trust-networking
+[End-to-end encryption]:      0001-high-level-architecture.html#user-goal-end-to-end-encryption
+[Physical portability]:       0001-high-level-architecture.html#user-goal-physical-portability
+[Commodity power and I/O]:    0001-high-level-architecture.html#user-goal-commodity-power-io
+
+### Guide-level explanation
+
+Explain the proposal as if it was already a feature of the project and this would be the documentation for that feature.
+
+- Introducing new named concepts.
```

Comment:
BMC and RMC would be two examples.

Reply:
This section needs a bit of a rewrite.

Reply:
Done in https://github.com/racklet/racklet/pull/20/commits/177c36949511bed9eac2607fbc8b39cad96605c1.